client/src/core/client/stream/tabs/Notifications/NotificationContainer.tsx

```diff
@@ -83,6 +86,13 @@ const getTitle = (
       "Your comment has been published"
     );
   }
+  if (type === GQLNOTIFICATION_TYPE.PREVIOUSLY_REJECTED_COMMENT_APPROVED) {
+    return getMessage(
+      bundles,
+      "notifications-yourPreviouslyRejectedCommentHasBeenApproved",
+      "Your comment was previously rejected. It has now been restored to the page."
```

Comment:
"restored to the page"

Have we checked this copy with Andrew?

Reply:
This is what was included in the issue. @losowsky can you confirm that this is the copy we want for this?

Reply:
Updated...thank you both!